docs/sbft-formal-model/replica.i.dfy

```diff
@@ -42,59 +42,80 @@ module Replica {
     map[]
   }
 
+  function NothingCommitted() : CommittedClientOperations {
+    map[]
+  }
+
+  // Define your Host protocol state machine here.
```

Comment:
nit - we can probably remove this comment?

Reply:
I will restructure the comments according to the context.